.gitignore

```diff
@@ -4,9 +4,8 @@ skema_venv/*
 *.pex
 *.pyc
 __pycache__/
-instance/*
+instance/log.txt
+instance/skemaDB.sqlite
```

Comment:
Is this necessary?  Seems like it is the opposite of what we want.  What would we want to keep in the instance folder?